--- origsrc/ctl-1.4.1/IlmCtlSimd/Makefile.am	2007-03-20 15:33:53.000000000 -0500
+++ src/ctl-1.4.1/IlmCtlSimd/Makefile.am	2007-10-28 22:34:49.437500000 -0500
@@ -96,7 +96,7 @@
 halfExpLogTable_LDFLAGS = -version-info @LIBTOOL_VERSION@ @ILMBASE_LDFLAGS@ \
 			  @ILMBASE_LIBS@ -no-undefined
 
-halfExpLog.h halfExpLogTable.h:	halfExpLogTable
+halfExpLog.h halfExpLogTable.h:	halfExpLogTable$(EXEEXT)
 	./halfExpLogTable  halfExpLogTable.h halfExpLog.h
 
 BUILT_SOURCES = halfExpLogTable.h
